src/spikeinterface/widgets/unit_waveforms.py

```diff
@@ -134,19 +133,19 @@ def __init__(
         else:
             if sparsity is None:
                 # in this case, we construct a dense sparsity
-                unit_id_to_channel_ids = {u: sorting_analyzer.channel_ids for u in sorting_analyzer.unit_ids}
+                unit_id_to_channel_ids = {u: channel_ids for u in sorting_analyzer.unit_ids}
```

Comment:
No. This will break the ipywidgets.
We need then all in case th user click on something diffrent.